test/WebSites/XmlFormattersWebSite/Startup.cs

```diff
@@ -72,6 +79,7 @@ public void Configure(IApplicationBuilder app)
                 routes.MapRoute("ActionAsMethod", "{controller}/{action}",
                     defaults: new { controller = "Home", action = "Index" });
 
+                routes.MapWebApiRoute("DefaultApi", "api/{controller}/{id?}");
```

Comment:
I am exercising a scenario where I am returning an IQueryable from an ApiController..its in `PeopleController`